contribute/hacker-guide.md

```diff
@@ -177,7 +177,7 @@ Listening for transport dt_socket at address: 8002
 // skip docs for local publishing
 publishArtifact in (Compile, packageDoc) in ThisBuild := false
 // set version based on current sha, so that you can easily consume this build from another sbt project
-baseVersionSuffix := s"local-${Process("tools/get-scala-commit-sha").lines.head.substring(0, 7)}"
+baseVersionSuffix := s"local-${VersionUtil.gitProperties.value.sha}"
```

Comment:
It seems you need to fully qualify `scala.build.VersionUtil`. Also, it's a good idea to start the suffix with `bin`, so that the version is considered binary compatible by sbt. For example, `2.12.3-bin-what-ever-you-want` has binary version `2.12` in sbt. So i'd suggest something like

```
baseVersionSuffix := s"bin-${scala.build.VersionUtil.gitProperties.value.sha}-local"
```